distribution/docs/src/main/resources/content/_sources/csw-federation-profile-source.adoc

```diff
@@ -11,28 +11,26 @@
 
 == {title}
 
-The CSW Federation Profile Source supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
+The CSW Federation Profile Source is ${branding}'s full fidelity CSW Federation Profile which supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
 
-Use the CSW source if querying a CSW version 2.0.2 compliant service.
+Use this when federating to a ${ddf-branding}-based system.
```

Comment:
How about this `this source type`or even ` Use the  CSW Federation Profile Source` instead of `this`? The reference is pretty far from the referent.

Reply:
fixed.